test/framework/easyblock.py

```diff
@@ -301,6 +301,51 @@ def test_make_module_extend_modpath(self):
             regex = re.compile(regex, re.M)
             self.assertTrue(regex.search(txt), "Pattern '%s' found in: %s" % (regex.pattern, txt))
 
+        # Repeat this but using an alternate envvars (instead of $HOME)
+        list_of_envvars = ['TURKEY', 'HAM']
+        os.environ['TURKEY'] = "1"
+        os.environ['HAM'] = "1"
```

Comment:
Use more sensible values and variable names here?

`1` doesn't make sense at all, make it `self.test_prefix` (since that's a path)?